packages/firestore/src/local/indexeddb_persistence.ts

```diff
@@ -105,15 +105,11 @@ const MAX_PRIMARY_ELIGIBLE_AGE_MS = 5000;
 const CLIENT_METADATA_REFRESH_INTERVAL_MS = 4000;
 /** User-facing error when the primary lease is required but not available. */
 const PRIMARY_LEASE_EXCLUSIVE_ERROR_MSG =
-  'Failed to obtain exclusive access to the persistence layer. ' +
-  'To allow shared access, make sure to invoke ' +
-  '`enablePersistence()` with `synchronizeTabs:true` in all tabs. ' +
-  'If you are using `experimentalForceOwningTab:true`, make sure that only ' +
-  'one tab has persistence enabled at any given time.';
+  'Failed to obtain exclusive access to the persistence layer. To allow ' +
+  'shared access, multi-tab synchronization has to be enabled in all tabs.';
```

Comment:
It seems a shame to remove the helpful error message regarding `experimentalForceOwningTab:true`, though it seems like we should know if that's enabled and give that helpful message only when applicable.

Reply:
I was hoping to not use the "experimental" name in firestore-exp and dropped it due to the two different names. I will come up with a way to add it back.